src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -28,7 +35,23 @@ export const CommitQueueCard: React.FC<Props> = ({
   commitTime,
   patchId,
   moduleCodeChanges,
+  commitQueueId,
 }) => {
+  const [removePatchFromCommitQueue, { loading }] = useMutation<
+    RemovePatchFromCommitQueueMutation,
+    RemovePatchFromCommitQueueMutationVariables
+  >(REMOVE_PATCH_FROM_COMMIT_QUEUE);
+  const handleEnroll = async (e) => {
+    e.preventDefault();
+    try {
+      await removePatchFromCommitQueue({
+        variables: { patchId, commitQueueId },
+        refetchQueries: ["CommitQueue"],
```

Comment:
I think it's a good practice to include the query variables for the refetch

Reply:
The Apollo docs say they are optional but if we want to adopt this practice i dont mind.  It does have drawbacks tho in that we will have to pass in the query variables into every refetchQueries instance and could involve passing  props to components just for the sake of  refetching the query. Up to you what do you think?

Reply:
@khelif96 that's a good point. let's leave it as is.